lib/mailserver.js

```diff
@@ -170,19 +170,25 @@ mailServer.create = function (
   mailDir,
   user,
   password,
-  hideExtensions
+  hideExtensions,
+  isSecure,
+  certFilePath,
+  keyFilePath
 ) {
   mailServer.mailDir = mailDir || defaultMailDir
   createMailDir()
 
   const hideExtensionOptions = getHideExtensionOptions(hideExtensions)
   const smtpServerConfig = Object.assign(
     {
+      secure: isSecure,
+      cert: certFilePath ? fs.readFileSync(certFilePath) : null,
+      key: keyFilePath ? fs.readFileSync(keyFilePath) : null,
```

Comment:
isn't maildev currently using self signed certs for this? Can we keep this behavior as a fallback?

Reply:
I don't think it does. From what I can tell from my understanding of the code, here's any support for SSL, self-signed or not, currently. So this shouldn't be breaking any existing configurations.